pkg/ccl/crosscluster/physical/alter_replication_job_test.go

```diff
@@ -68,6 +65,8 @@ func TestAlterTenantCompleteToLatest(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	skip.UnderStress(t, "flaky test")
```

Comment:
hrm, did this pr cause this test to be flakey? if so, we should figure out why.

Reply:
Ah this should've been removed, I skipped the test at some point but the test now passes under stress when I tested it locally and through extended CI.